File: Selenium/Activities/Activity_19.java
package Activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_19 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("The Page title is: " + driver.getTitle());
        driver.findElement(By.id("confirm")).click();
        Alert alert = driver.switchTo().alert();
        System.out.println("The alert message is: " + alert.getText());
        alert.accept();

        // for dismiss
        driver.findElement(By.id("confirm")).click();
        Alert alert1 = driver.switchTo().alert();
        System.out.println("The alert message is: " + alert1.getText());

        alert1.dismiss();

        driver.quit();
    }
}

File: Selenium/Activities/Activity_19.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("The page title is: ", driver.title)
    driver.maximize_window()
    driver.find_element(By.ID, "confirm").click()
    alert = driver.switch_to.alert
    print(alert.text)
    time.sleep(5)
    alert.accept()


    # for dismiss
    driver.find_element(By.ID, "confirm").click()
    alert = driver.switch_to.alert
    time.sleep(5)
    alert.dismiss()

    driver.quit()
